### PR TITLE
first commit on toggle superuser.

### PR DESCRIPTION
**Pull request by Jan-M**
make superuser not the default for team members

### CODE REVIEW

File: pkg/util/config/config.go [hunk 1]
```diff
@@ -58,6 +58,7 @@ type Config struct {
 	DebugLogging          bool           `name:"debug_logging" default:"true"`
 	EnableDBAccess        bool           `name:"enable_database_access" default:"true"`
 	EnableTeamsAPI        bool           `name:"enable_teams_api" default:"true"`
+	EnableTeamsSuperuser  bool           `name:"enable_teams_superuser" default: "false"`
```
Comment (ikitiki) [inline pkg/util/config/config.go]: space between ``:`` and ``"``

**Comment by ikitiki** [comment]
I'd edit config map as well

File: pkg/cluster/cluster.go [hunk 2]
```diff
@@ -644,7 +644,11 @@ func (c *Cluster) initHumanUsers() error {
 		return fmt.Errorf("could not get list of team members: %v", err)
 	}
 	for _, username := range teamMembers {
-		flags := []string{constants.RoleFlagLogin, constants.RoleFlagSuperuser}
+		flags := []string{constants.RoleFlagLogin}
+		if c.OpConfig.EnableTeamsSuperuser {
+			flags = flags.append(constants.RoleFlagSuperuser)
```
Comment (ikitiki) [inline pkg/cluster/cluster.go]: ``flags = append(flags, constants.RoleFlagSuperuser)``

**Comment by ikitiki** [comment]
please remove ``.idea/*`` from the PR

**Comment by ikitiki** [comment]
add ``enable_teams_superuser`` with default value to the ``configmap.yaml`` as well

File: manifests/configmap.yaml [hunk 3]
```diff
@@ -27,7 +27,9 @@ data:
   resource_check_timeout: 10m
   resync_period: 5m
   super_username: postgres
-  teams_api_url: http://fake-teams-api.default.svc.cluster.local
+  enable_teams_api: "false"
+  enable_teams_superuser: "false"
+  # teams_api_url: http://fake-teams-api.default.svc.cluster.local
```
Comment (ikitiki) [inline manifests/configmap.yaml]: maybe we should leave it uncommented

File: pkg/cluster/cluster.go [hunk 4]
```diff
@@ -644,8 +644,18 @@ func (c *Cluster) initHumanUsers() error {
 		return fmt.Errorf("could not get list of team members: %v", err)
 	}
 	for _, username := range teamMembers {
-		flags := []string{constants.RoleFlagLogin, constants.RoleFlagSuperuser}
+		flags := []string{constants.RoleFlagLogin}
 		memberOf := []string{c.OpConfig.PamRoleName}
+
+		if c.OpConfig.EnableTeamSuperuser {
+			flags = append(flags, constants.RoleFlagSuperuser)
+		}
```
Comment (ikitiki) [inline pkg/cluster/cluster.go]: ``} else {``

**Comment by Jan-M** [comment]
:+1:

**Comment by ikitiki** [comment]
👍

**Comment by ikitiki** [comment]
don't forget to squash the commits

**Pull request merged**